src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/ConstructorWriter.java

```diff
@@ -106,13 +106,13 @@ protected void buildConstructorDoc(Content target) {
             for (Element constructor : constructors) {
                 currentConstructor = (ExecutableElement)constructor;
                 Content constructorContent = getConstructorHeaderContent(currentConstructor);
-
-                buildSignature(constructorContent);
-                buildDeprecationInfo(constructorContent);
-                buildPreviewInfo(constructorContent);
-                buildConstructorComments(constructorContent);
-                buildTagInfo(constructorContent);
-
+                Content scrollBox = HtmlTree.DIV(HtmlStyle.horizontalScroll);
```

Comment:
Stylistically, it's a shame to change the name of the variable in this and similar situations when all you are changing is its initial value. In the lines that follow, its only important that it is some kind of container, not specifically a `scrollBox`.  This is the variable name equivalent of whether to write
```
Map m = new HashMap();
```
or 
```
HashMap m = new HashMap();
```